data/hai/unfettered side missions.txt

```diff
@@ -65,3 +66,403 @@ mission "Unfettered returning home"
+			`	You see the Unfettered have already begun their launching procedures, so you should prepare for takeoff in your Anomalocaris, and no other ships, as soon as you can.`
+				accept
+			label headingout
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				set "unfettered: fled with the relic"
+			`	Seeing as they are unlikely to cooperate, you quickly head to your ship and launch, with the Unfettered rushing after you shortly after.`
+				launch
+	on enter
+		conversation
+			branch fleeing
+				has "unfettered: fled with the relic"
+			branch "too many ships"
+				"total ships" > 1
+			`As you leave the planet's atmosphere, you see your challenger already waiting for you in the system, and they send you a hail as soon as you are in space. "Show us what you've got, monkey! Show us that you control your weapons well, as any warrior should! Prove to us that you have the strength to disable our ships, yet still have the restraint not to destroy them!"`
+				accept
+			label fleeing
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				fail
+			`The Unfettered don't seem too pleased to see you leaving with their prize. The challengers are already waiting for you in orbit, and they hail you fast. "Where do you think you're going, monkey?"`
+				decline
+			label "too many ships"
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				fail
+			`As you take to join your fleet in orbit, you receive a hail from the Unfettered. "If you can't even count how many ships you have, you're not worthy of our time, monkey! Dishonor will get you nowhere with us True Hai!"`
+	on disabled
+		"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+		fail
+		conversation
+			`The enemy ship hails you after disabling your own. "Thought you could get away from us so easily? You better give us a good bribe to land on our worlds and return that ship, before we value blowing you up more than preserving it!"`
+				to display
+					has "unfettered: fled with the relic"
+				decline
+			`After your ship is disabled, one of the Unfettered ships hails you. "You have lost the challenge, but we didn't expect much from a human like you. We will repair your ship for a fair bribe, but you better land at one of our worlds and return it to us with a most generous compensation. If you try to keep it from us, we'll do things the hard way.`
+			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us. Maybe someday you will prove yourself otherwise."`
+	on fail
+		"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
```

Comment:
```suggestion
			`After your ship is disabled, one of the Unfettered ships hails you. "You have lost the challenge, but we didn't expect much from a human like you. We will repair your ship for a fair bribe, but you better land at one of our worlds and return it to us with a most generous compensation. If you try to keep it from us, we'll do things the hard way.`
			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us. Maybe someday you will prove yourself otherwise."`
	on visit
		fail
	on fail
		"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
```

Reply:
This is a `fail` now, so it'll go to the `on fail` conversation.

Reply:
Don't those only trigger the next time you land? I can test this tomorrow to see